### DISCUSSION TITLE
Markdown content in an HTML block is not parsed as Markdown...

### DISCUSSION THREAD

**Discussion by Enn83** [squidfunk/mkdocs-material#4641] [Q&A] [answered]
Hi!

I try to add something like this in an `*.md` file:

```
Markdown content and new lines ...

<table>
  <thead>
    <tr>
      <th>Head1</th>
      <th>Head2</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        CELL 1
        <br/><small>(:material-grid:{.shade75} ***Quelconque***)</small>
      </td>
      <td>
        :material-regex: **Regex :** ``regex``
        <br/> :material-pen: **Exemples :** `example`
      </td>
  </tbody>
</table>

New lines and Markdown content ...
```

I also add the `md_in_html` in the `markdown_extensions:` key in `mkdocs.yml`, but the HTML content is not parsed as markdown...

I try adding `markdown="1"`, `markdown="block"`, `markdown="span"`,  and `markdown` in HTML tags but nothing change...

```
<table>
  <thead>
    <tr>
      <th>Head1</th>
      <th>Head2</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td markdown="1">
        CELL 1
        <br/><small>(:material-grid:{.shade75} ***Quelconque***)</small>
      </td>
      <td markdown>
        :material-regex: **Regex :** ``regex``
        <br/> :material-pen: **Exemples :** `example`
      </td>
  </tbody>
</table>
```

Thank you in advance for your help!

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. @facelessuser any idea? I've fiddled around adding `markdown` attributes as well, no luck so far.

**Reply by facelessuser** [reply]
I don't know if it is clearly expressed in the Python Markdown documentation, but the content cannot be indented under child elments. It parses content on the document root level. So, HTML blocks are required to be on the root level and each start and end of the block element in which you which to parse content must be on its own line.

```html
<table markdown="1">
<tbody markdown="1">
<tr markdown="1">
<td markdown="1">
content **bold**
</td>
<td markdown="1">
content **bold**
</td>
</tbody>
</table>
```

**Reply by Enn83** [reply]
Thanks for your fast reply! 

I remember trying this one (adding `markdown="1"` in all HTML blocks from the root one up to the one I need to parse markdown syntax) but I'm not sure if each element was on its own line...

I'll try that as soon as possible and post updates soon!

**Reply by facelessuser** [reply]
It's particularly relevant to non-inline elements that are not normally parsed by Python Markdown. Inline elements that Python Markdown does recognize do not have to follow this. I don't think Python Markdown handles `table`, `tbody`, `tr`, `th`, or `td` (IIRC), so all of these I think require `markdown="1"`.

**Reply by Enn83** [reply]
I try that and it works fine!

I remove all indentation in left side, non-inline all markdown syntax with an HTML block start or end, and add the `markdown="1"` HTML tags on all intermediate HTML block.

Thank you very much!
